### DISCUSSION TITLE
cycle through the playback devices press with a hotkey like the soundswitch application does...

### DISCUSSION THREAD

**Discussion by bigplayer-ai** [File-New-Project/EarTrumpet#1293] [Ideas]
Hey,
Can you add this feature to eartrumpet?
https://github.com/Belphemur/SoundSwitch

**Comment by halaczkiewiczk** [comment]
There are more features on the [dev release](https://install.eartrumpet.app/dev/EarTrumpet.Package.appinstaller)
